lib/parameter_parser/search_parameter_parser.rb

```diff
@@ -458,15 +458,8 @@ def debug_options
   def ab_tests
     @ab_tests ||=
       begin
-        variants = character_separated_param("ab_tests")
-        variants = variants.map { |variant| variant.split(":", 2) }
-
-        variants.each_with_object({}) do |(variant_name, variant_code), variants_hash|
-          if variant_code.blank?
-            @errors << %{Invalid ab_tests, missing type "#{variant_name}"}
-          end
-          variants_hash[variant_name.to_sym] = variant_code
-        end
+        character_separated_param("ab_tests")
```

Comment:
Does this do anything?

Reply:
Yes, it stops the parser complaining about an unused parameter